pkgs/development/haskell-modules/patches/hie-build-plugin-haddock.patch

```diff
@@ -0,0 +1,23 @@
+commit ae0fb7a79a8912cf7a40bee418db239a7188805c
```

Comment:
Ditto. Can this file be downloaded from somewhere else?

Reply:
This is from a PR, I'll fetch it